settings.gradle

```diff
@@ -16,16 +16,25 @@
 
 rootProject.name="fiat"
 
+def plugins = [
+  ':fiat-plugins',
+  ':fiat-plugins:kork-secrets-aws',
+  ':fiat-plugins:kork-stackdriver',
```

Comment:
I assume these bits about `kork-*` would go away once kork is plugin-ified? Not sure I like each service having kork plugins defined in their codebase.